Resources/Textures/Structures/Windows/reinforced_uranium_window.rsi/meta.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "made by AsikKEsel",
```

Comment:
Isn't there existing sprites in ss13 to use instead? Also these are derivatives but don't credit the original source.